### ISSUE TITLE
Regarding validation ratio taken for split-ImageNet

### ISSUE THREAD

**Issue by pavanteja295** [facebookresearch/Adversarial-Continual-Learning#6]
In the paper the number of validation samples not used for training is  mentioned to be 375 out of  2500 per task. Which accounts to 15% of the training samples while in the code it is mentioned as 2% (pc_valid = 0.02). Can you please let me know which is the final validation ratio?

**Comment by SaynaEbrahimi** [comment]
For miniImageNet experiment the validation set is [0.02](https://github.com/facebookresearch/Adversarial-Continual-Learning/blob/4474bc02122998bb74202ac0bcc90cc0e5024f62/src/configs/config_miniimagenet.yml#L41) but for the rest of the experiment it is 0.15. The config file for each experiment is given [here](https://github.com/facebookresearch/Adversarial-Continual-Learning/tree/master/src/configs) and those are the exact numbers used to get the results reported in the paper. You can replicate those with the commands given in the[ README.md file](https://github.com/facebookresearch/Adversarial-Continual-Learning/blob/master/README.md).
